### DISCUSSION TITLE
Gaps Specific to workspace

### DISCUSSION THREAD

**Discussion by wesbos** [nikitabobko/AeroSpace#707] [questions-and-answers]
I know we can have [gaps specific to monitors](https://nikitabobko.github.io/AeroSpace/guide#key-mapping) and I know we can [assign workspaces to monitors](https://nikitabobko.github.io/AeroSpace/guide#assign-workspaces-to-monitors), 

I'm looking for a increase the gaps on a single workspace only. The reason behind is is that I record my screen for videos but need to only record a sub-set of the screen. Having larger gaps on the outer will allow me to adjust them so the windows fit in the recording area.

**Comment by alewis001** [comment]
I think I'd like to see this as well. 

I have a 4k monitor and I like to have the browser and other windows in a central portion of the screen (which I achieve using gaps to get the right size and position); however, there are occasions where I'd like to have some windows side-by-side using all of the screen.

Currently, if I have the side-by-side windows as floating, when moving through the tree it keeps flip-flopping to the floating windows; I assume because of the overlap algorithm. I have therefore created a separate workspace to put the side-by-side windows which are floating so they ignore the monitor `gaps` setting but then navigation using `focus` doesn't work (maybe because there it no root container that "overlaps"?).

Maybe it's the flip-flopping between the floating and non-floating which is really my issue and maybe there is a better/different way to handle that?

It does also get complicated as I'm also on a laptop with an external screen. As such, when I unplug the external monitor I effectively want everything to be maximised on the laptop screen (no gaps). I have the "with" and "without" external monitor setups working right now.

**Comment by besseddrest** [comment]
+1 here

**Comment by r00tnix** [comment]
+1 As well, would be nice to have one workspace which moves over just enough to show the few icons on the desktop, but the rest of the workspaces are full screen
